functions/createEvent.js

```diff
@@ -0,0 +1,35 @@
+const { db } = require('./admin');
+
+const createEvent = () => {
+//    if (checkInput(request, response) === true) {
```

Comment:
om du ids kan du ju ta bort bortkommenterad kod!

Reply:
Fixed in next version